cpp/src/parquet/CMakeLists.txt

```diff
@@ -171,6 +171,9 @@ set(PARQUET_SRCS
     exception.cc
     file_reader.cc
     file_writer.cc
+    geospatial_statistics.cc
```

Comment:
I don't remember if it was already discussed or not, but do we want to put these files in a `geospatial` subdirectory? In the future we might want to expose more utilities there to deal with geo data (or perhaps not?).

Reply:
Yes, I think it would be cleaner in a dedicated subdirectory (for example `geospatial/statistics.cc`).

We could do the same for the various `level_` files at some point :)